src/website/views/utilities/flex.pug

```diff
@@ -197,10 +198,10 @@ h4 Center
 h4 Baseline
 .example.-mb--3
   .-p--3.-pb--1
-    .-mb--2.-d--flex(style="background:#F2FAE6;height:7rem;")
-      .-text.-p--2.-b--1(style="background:#E4F2CE;border-color:#C6DBA7;") Item
-      .-text.-p--2.-b--1.-align-self--baseline(style="background:#E4F2CE;border-color:#C6DBA7;") Aligned Item
-      .-text.-p--2.-b--1(style="background:#E4F2CE;border-color:#C6DBA7;") Item
+    .-mb--2.-d--flex.-show--example(style="background:#EDFAF3;height:7rem;")
```

Comment:
@davidojedacl There's a bit of a color mixup here. The background color is pointing to the mint/emerald palette and the pseudo elements are pointing to green. Let's replace the inline `#EDFAF3` backgrounds with the utility class `-bg--grey-10`.

Reply:
ok